tsconfig.json

```diff
@@ -5,10 +5,11 @@
     "importHelpers": true,
     "lib": ["es2018"],
     "module": "commonjs",
+    "noImplicitAny": false,
     "outDir": "build",
     "rootDir": "src",
     "skipLibCheck": true,
-    "strict": true,
+    "strict": false,
```

Comment:
I think `strict` should be true, this way unnused variables, for example, would result in compilation error

Reply:
But this depend on the amount of errors this will trigger, if it's too much it's ok, but if so, add the flags:
```
    "noUnusedLocals": true,
    "noUnusedParameters": true,
```

Reply:
The plugin template should have strict set to true

Reply:
With `"strict": true,` this plugin get this error

![](https://i.imgur.com/sK340xA.png)

Reply:
this can be solved with a `!` following the app - this is a typescript operator that tells the compiler that you know what you're doing and in this case the variable is not null/undefined -> `app!`

Reply:
(not such a good practice to use it though, but in this case it's ok, it's a minor technical debt that we are adding to it)

Reply:
Sure, I got it!